msteams-platform/TOC.yml

```diff
@@ -20,136 +14,26 @@
-  - name: Tool options and code samples
-    href: get-started/tool-options-and-code-samples.md
-    displayName: Teams app samples, tools to build Teams app
-- name: Plan your app or agent
-  items:
-  - name: Overview
-    href: concepts/app-fundamentals-overview.md
-    displayName: Teams app features, Government Community Cloud (GCC), Department of Defense (DoD), GCC High (GCCH), Teams operated by 21Vianet
-  - name: Understand use cases
-    href: concepts/design/understand-use-cases.md
-    displayName: user requirements, app functionalities, app scope, capabilities
-  - name: Map use case to Teams features
-    href: concepts/design/map-use-cases.md
-    displayName: user scenarios, Teams app features
-  - name: Teams app planning checklist
-    href: concepts/design/planning-checklist.md
-    displayName: HTTPs, localhost, host your Teams app
-  - name: Plan responsive tabs for Teams mobile
-    href: concepts/design/plan-responsive-tabs-for-teams-mobile.md
-    displayName: tab app in Teams mobile, Teams mobile apps
-  - name: Plan analytics for your Teams app
-    items:
-      - name: Get started with app analytics
-        href: concepts/design/overview-analytics.md
-        displayName: data instrumentation, user-specific insights, constructs, conversational, hosted web canvas 
-      - name: Strategize and decide what to measure
-        href: concepts/design/strategize-measure.md
-        displayName: measure your products, monitor events, product satisfaction, customer values
-      - name: Plan code instrumentation using analytics service
-        href: concepts/design/plan-code-instrumentation.md
-        displayName: track events, monitor, Azure Applications Insights
-  - name: Plan to send app notifications
-    href: concepts/design/design-app-notification.md
-    displayName: notification framework, activity feed, message, alert, notify
-  - name: Plan for special clouds
-    items:
-    - name: Plan for government clouds
-      href: concepts/cloud-overview.md
-      displayName: Government Community Cloud (GCC), GCC High (GCCH), Department of Defense (DoD), Teams for government tenant
-    - name: Plan for sovereign clouds
-      href: concepts/sovereign-cloud.md
-      displayName: Teams operated by 21Vianet, gallatin
-- name: Design your app or agent
+- name: What's new?
+  href: whats-new.md
+  displayName: Teams platform features, latest updates, RSS feed, deprecated, announcement, preview
+- name: Glossary
+  href: get-started/glossary.md
+  displayName: App tray, terminologies, definitions, welcome, new word, meaning
+- name: What are agents in Teams
+  href: /microsoft-365-copilot/extensibility/overview-custom-engine-agent?toc=/microsoftteams/platform/toc.json&bc=/microsoftteams/platform/breadcrumb/toc.json
+  displayName: AI bot, Teams AI library, custom agent
+- name: Build your first agent
+  href: bots/how-to/teams-conversational-ai/conversation-ai-quick-start.md
```

Comment:
I can't find the target topic for this TOC link. I assume that it's about building a CEA? What tool does the quick start use here? I would be curious to understand how Build your first... is different than Build with Teams AI library, and also the Tools and SDKs section further down.

You might want to consider a TOC node for "Build agents", that might be structured like this:

Build agents
    Build your first agent
    Build agents with Teams AI library
    Tools
    SDKs

Reply:
This link doesn't exist. 
`/microsoftteams/platform/bots/how-to/teams-conversational-ai/conversation-ai-quick-start?branch=pr-en-us-13218`

And redirect to `teams-ai-library/welcome?branch=main`(fallback to the main branch).